src/components/Button/Button.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import style from './Button.st.css';
+import {ButtonNext} from 'wix-ui-core/button-next';
+import {ButtonProps as ButtonNextProps} from 'wix-ui-core/dist/src/components/button-next/button-next';
+// import { withEllipsedTooltip } from 'wix-ui-core/dist/src/hocs/EllipsedTooltip';
+
+export enum PRIORITY {
+  primary = 'primary',
+  secondary = 'secondary'
+}
+
+export interface ButtonProps extends ButtonNextProps {
+  dataHook?: string;
```

Comment:
because of ButtonNext and it's UniDriver i had to add the dataHook
is there any better solution?

Reply:
why is that? this should not be a mandatory property

Reply:
https://github.com/wix/wix-ui/blob/master/packages/wix-ui-test-utils/src/vanilla/vanilla.tsx#L59
because of this line i get props error in Button

Reply:
@shlomitc after new install of node modules i stopped getting the error. i'll delete it